platform/marvell/sonic-platform-nokia/debian/sonic-platform-nokia-7215.postinst

```diff
@@ -10,7 +10,7 @@ systemctl enable nokia-7215init.service
 systemctl start nokia-7215init.service
 
 systemctl enable cpu_wdt.service
-systemctl start cpu_wdt.service
+systemctl start cpu_wdt.service &
```

Comment:
If this line is still needed, please use the `--no-block` argument instead of backgrounding this job.